TestPlaybooks/NonCircleTests/Deprecated/playbook-DocumentationTest.yml

```diff
@@ -55,7 +56,7 @@ tasks:
         simple: "yes"
       unsecure: {}
       url:
-        simple: https://github.com/demisto/content/raw/master/Packs/ipinfo/Integrations/integration-Ipinfo.yml
+        simple: https://github.com/demisto/content/raw/master/Packs/ipinfo/Integrations/ipinfo.yml
```

Comment:
please change to `integration-ipinfo.yml` and not `ipinfo.yml`

Reply:
done